features/support/env.rb

```diff
@@ -26,11 +26,15 @@
 
 Capybara.server = :webrick
 
-Capybara.register_driver :poltergeist do |app|
-  Capybara::Poltergeist::Driver.new(app, timeout: 30)
+Capybara.register_driver :apparition do |app|
+  options = {
+    headless: true, skip_image_loading: false, window_size: [1366, 768], timeout: 30,
+    browser_options: [:disable_gpu, :no_sandbox, disable_features: 'VizDisplayCompositor']
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.